orientdb/src/main/java/com/yahoo/ycsb/db/OrientDBClient.java

```diff
@@ -80,30 +80,26 @@
   private static final String ORIENTDB_MASSIVEREAD = "massiveread";
   private static final String ORIENTDB_NOCACHE = "nocache";
 
-  private final Logger log = LoggerFactory.getLogger(getClass());
-
-  @Override
-  public void init() throws DBException {
-    Properties props = getProperties();
+  private static final Logger LOG = LoggerFactory.getLogger(OrientDBClient.CLASS);
 
+  private static synchronized ODatabaseDocumentTx initDB(Properties props) throws DBException {
```

Comment:
Please add javadoc explaining why this needs to be synchronized (it's not obvious because the various clients don't appear to share connection objects).

I think it's because of the automatic table creation?